<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<meta name="description" content="Personal Website for Preonath Chondrow Dev (Bioinformatician, Data Analyst)">
<title>Resources | Preonath</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>

<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href=".//files/profiles/icon.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-de84f8d6bb715db06a919283c2d1e787.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-abb4c31c7d01e939dc93a8370005825e.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light"><script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<meta property="og:title" content="Resources | Preonath">
<meta name="twitter:title" content="Resources | Preonath">
<meta name="twitter:card" content="summary">
</head>
<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top"><nav class="navbar navbar-expand-lg " data-bs-theme="dark"><div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
<li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./research.html"> 
<span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./publications.html"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./conference.html"> 
<span class="menu-text">Conference</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./training.html"> 
<span class="menu-text">Training</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./teaching.html"> 
<span class="menu-text">Teaching</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./skill.html"> 
<span class="menu-text">Skills</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./cv/index.html"> 
<span class="menu-text">CV</span></a>
  </li>  
</ul>
<ul class="navbar-nav navbar-nav-scroll ms-auto"></ul>
</div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav></header><!-- content --><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active"><h2 id="toc-title">On this page</h2>
   
  <ul>
<li>
<a href="#career-topics" id="toc-career-topics" class="nav-link active" data-scroll-target="#career-topics">Career Topics</a>
  <ul class="collapse">
<li><a href="#general" id="toc-general" class="nav-link" data-scroll-target="#general">General</a></li>
  <li><a href="#individual-development-plan-idp" id="toc-individual-development-plan-idp" class="nav-link" data-scroll-target="#individual-development-plan-idp">Individual Development Plan (IDP)</a></li>
  <li><a href="#postdoc-related" id="toc-postdoc-related" class="nav-link" data-scroll-target="#postdoc-related">Postdoc Related</a></li>
  <li><a href="#data-science-internships" id="toc-data-science-internships" class="nav-link" data-scroll-target="#data-science-internships">Data Science Internships</a></li>
  <li><a href="#academic-job-searchhiring" id="toc-academic-job-searchhiring" class="nav-link" data-scroll-target="#academic-job-searchhiring">Academic Job Search/Hiring</a></li>
  <li><a href="#academic-career" id="toc-academic-career" class="nav-link" data-scroll-target="#academic-career">Academic Career</a></li>
  </ul>
</li>
  <li>
<a href="#teaching-tools" id="toc-teaching-tools" class="nav-link" data-scroll-target="#teaching-tools">Teaching Tools</a>
  <ul class="collapse">
<li><a href="#software" id="toc-software" class="nav-link" data-scroll-target="#software">Software</a></li>
  <li><a href="#books" id="toc-books" class="nav-link" data-scroll-target="#books">Books</a></li>
  <li><a href="#online-resources" id="toc-online-resources" class="nav-link" data-scroll-target="#online-resources">Online Resources</a></li>
  <li><a href="#other" id="toc-other" class="nav-link" data-scroll-target="#other">Other</a></li>
  </ul>
</li>
  <li>
<a href="#programming" id="toc-programming" class="nav-link" data-scroll-target="#programming">Programming</a>
  <ul class="collapse">
<li><a href="#r-programming" id="toc-r-programming" class="nav-link" data-scroll-target="#r-programming">R Programming</a></li>
  <li><a href="#lists-by-others" id="toc-lists-by-others" class="nav-link" data-scroll-target="#lists-by-others">Lists by others</a></li>
  <li><a href="#online-courses-for-r" id="toc-online-courses-for-r" class="nav-link" data-scroll-target="#online-courses-for-r">Online Courses for R</a></li>
  <li><a href="#r-books-and-tutorials" id="toc-r-books-and-tutorials" class="nav-link" data-scroll-target="#r-books-and-tutorials">R books and Tutorials</a></li>
  <li><a href="#general-r-online-resources" id="toc-general-r-online-resources" class="nav-link" data-scroll-target="#general-r-online-resources">General R Online Resources</a></li>
  <li><a href="#r-and-shiny" id="toc-r-and-shiny" class="nav-link" data-scroll-target="#r-and-shiny">R and Shiny</a></li>
  <li><a href="#r-markdown-and-related" id="toc-r-markdown-and-related" class="nav-link" data-scroll-target="#r-markdown-and-related">R Markdown and related</a></li>
  <li><a href="#r-package-development" id="toc-r-package-development" class="nav-link" data-scroll-target="#r-package-development">R package development</a></li>
  </ul>
</li>
  <li>
<a href="#data-analysis-and-statistics" id="toc-data-analysis-and-statistics" class="nav-link" data-scroll-target="#data-analysis-and-statistics">Data Analysis and Statistics</a>
  <ul class="collapse">
<li><a href="#other-lists" id="toc-other-lists" class="nav-link" data-scroll-target="#other-lists">Other lists</a></li>
  </ul>
</li>
  <li><a href="#online-courses-for-data-analysis" id="toc-online-courses-for-data-analysis" class="nav-link" data-scroll-target="#online-courses-for-data-analysis">Online Courses for Data Analysis</a></li>
  <li><a href="#data-analysis-books" id="toc-data-analysis-books" class="nav-link" data-scroll-target="#data-analysis-books">Data Analysis Books</a></li>
  <li>
<a href="#specific-topics" id="toc-specific-topics" class="nav-link" data-scroll-target="#specific-topics">Specific Topics</a>
  <ul class="collapse">
<li><a href="#data-cleaning-and-checking" id="toc-data-cleaning-and-checking" class="nav-link" data-scroll-target="#data-cleaning-and-checking">Data cleaning and checking</a></li>
  <li><a href="#visualization" id="toc-visualization" class="nav-link" data-scroll-target="#visualization">Visualization</a></li>
  <li><a href="#missing-dataimputation" id="toc-missing-dataimputation" class="nav-link" data-scroll-target="#missing-dataimputation">Missing Data/Imputation</a></li>
  <li><a href="#p-values" id="toc-p-values" class="nav-link" data-scroll-target="#p-values">p-values</a></li>
  <li><a href="#epidemiology-specific-topics" id="toc-epidemiology-specific-topics" class="nav-link" data-scroll-target="#epidemiology-specific-topics">Epidemiology specific topics</a></li>
  <li><a href="#time-series-analysis" id="toc-time-series-analysis" class="nav-link" data-scroll-target="#time-series-analysis">Time-series analysis</a></li>
  </ul>
</li>
  <li>
<a href="#data-sources" id="toc-data-sources" class="nav-link" data-scroll-target="#data-sources">Data Sources</a>
  <ul class="collapse">
<li><a href="#general-data-related-resources" id="toc-general-data-related-resources" class="nav-link" data-scroll-target="#general-data-related-resources">General Data Related Resources</a></li>
  <li><a href="#some-data-sources" id="toc-some-data-sources" class="nav-link" data-scroll-target="#some-data-sources">Some data Sources</a></li>
  <li>
<a href="#infectious-disease-specific" id="toc-infectious-disease-specific" class="nav-link" data-scroll-target="#infectious-disease-specific">Infectious Disease Specific</a>
  <ul class="collapse">
<li><a href="#general-1" id="toc-general-1" class="nav-link" data-scroll-target="#general-1">General</a></li>
  <li><a href="#influenza" id="toc-influenza" class="nav-link" data-scroll-target="#influenza">Influenza</a></li>
  <li><a href="#tb" id="toc-tb" class="nav-link" data-scroll-target="#tb">TB</a></li>
  </ul>
</li>
  </ul>
</li>
  <li>
<a href="#data-and-tools-for-cancer-genomics" id="toc-data-and-tools-for-cancer-genomics" class="nav-link" data-scroll-target="#data-and-tools-for-cancer-genomics">Data and Tools for Cancer Genomics</a>
  <ul class="collapse">
<li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a></li>
  <li><a href="#tools" id="toc-tools" class="nav-link" data-scroll-target="#tools">Tools</a></li>
  <li><a href="#guideline-for-bioconductor-users" id="toc-guideline-for-bioconductor-users" class="nav-link" data-scroll-target="#guideline-for-bioconductor-users">Guideline for Bioconductor Users</a></li>
  <li><a href="#required-software" id="toc-required-software" class="nav-link" data-scroll-target="#required-software">Required software</a></li>
  <li><a href="#prework" id="toc-prework" class="nav-link" data-scroll-target="#prework">Prework</a></li>
  <li><a href="#install-bioconductor-packages" id="toc-install-bioconductor-packages" class="nav-link" data-scroll-target="#install-bioconductor-packages">Install Bioconductor Packages</a></li>
  <li><a href="#r-packages-rnaseq-and-single-cell-rna-seq-analysis" id="toc-r-packages-rnaseq-and-single-cell-rna-seq-analysis" class="nav-link" data-scroll-target="#r-packages-rnaseq-and-single-cell-rna-seq-analysis">R Packages RNASeq and Single-cell RNA-seq Analysis</a></li>
  <li><a href="#blogs-for-r-programming-statistics-and-data-analyis" id="toc-blogs-for-r-programming-statistics-and-data-analyis" class="nav-link" data-scroll-target="#blogs-for-r-programming-statistics-and-data-analyis">Blogs for R Programming, Statistics, and Data Analyis</a></li>
  <li><a href="#videos" id="toc-videos" class="nav-link" data-scroll-target="#videos">Videos</a></li>
  </ul>
</li>
  </ul></nav>
    </div>
<!-- main -->
<main class="content column-page-left" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block default"><div class="quarto-title">
<h1 class="title">Resources</h1>
</div>



<div class="quarto-title-meta column-page-left">

    
  
    
  </div>
  


</header><section id="career-topics" class="level1"><h1>Career Topics</h1>
<section id="general" class="level2"><h2 class="anchored" data-anchor-id="general">General</h2>
<ul>
<li><p><a href="https://www.insidehighered.com/career-advice/carpe-careers">(Academic) career related essays</a></p></li>
<li><p><a href="http://www.nature.com/scitable/topic/career-planning-14121550">A collection of resources on (scientific) career planning</a> from the Nature Publishing Group.</p></li>
<li><p>A resource from the NIH toward <a href="http://www.nihbest.org/">Broadening Experiences in Scientific Training</a></p></li>
<li><p>A good list of books on <a href="https://fastertomaster.com/ultimate-learning-how-to-learn-reading-list/">‘learning how to learn’</a></p></li>
</ul></section><section id="individual-development-plan-idp" class="level2"><h2 class="anchored" data-anchor-id="individual-development-plan-idp">Individual Development Plan (IDP)</h2>
<ul>
<li>
<a href="https://myidp.sciencecareers.org/">myIDP from AAAS.</a> Free IDP focused on scientific career planning.</li>
</ul></section><section id="postdoc-related" class="level2"><h2 class="anchored" data-anchor-id="postdoc-related">Postdoc Related</h2>
<ul>
<li><a href="https://www.insidehighered.com/advice/2016/08/23/should-you-pursue-postdoc-or-not-essay">Thoughts on doing a postdoc or not</a></li>
</ul></section><section id="data-science-internships" class="level2"><h2 class="anchored" data-anchor-id="data-science-internships">Data Science Internships</h2>
<ul>
<li>
<a href="http://www.triageconsulting.com/">Insight Data Science</a> arranges short internships with some of the big names.</li>
</ul></section><section id="academic-job-searchhiring" class="level2"><h2 class="anchored" data-anchor-id="academic-job-searchhiring">Academic Job Search/Hiring</h2>
<ul>
<li>
<a href="https://chroniclevitae.com/news/1599-reading-your-application-file">This blog post provides a nice description how applications for faculty positions are reviewed</a>. From my own experience having been on and chaired several faculty searches, I think it’s a fairly accurate description.</li>
</ul></section><section id="academic-career" class="level2"><h2 class="anchored" data-anchor-id="academic-career">Academic Career</h2>
<ul>
<li><a href="https://www.castoredc.com/blog/50-actionable-tips-to-help-you-survive-your-phd/">Tips to have a productive PhD</a></li>
</ul></section></section><section id="teaching-tools" class="level1"><h1>Teaching Tools</h1>
<section id="software" class="level2"><h2 class="anchored" data-anchor-id="software">Software</h2>
<ul>
<li>
<a href="https://rstudio.github.io/learnr/">learnr</a> - R package that allows development of interactive web-based R tutorials.</li>
<li>
<a href="https://minecr.shinyapps.io/feedback-at-scale/">Feedback at scale</a> - tutorial for using <code>learnr</code> and <code>gradethis</code> as teaching tools.</li>
</ul></section><section id="books" class="level2"><h2 class="anchored" data-anchor-id="books">Books</h2>
<ul>
<li>
<a href="https://rstudio4edu.github.io/rstudio4edu-book/">rstudio4edu</a> - A Handbook for Teaching and Learning with R and RStudio - still very much a draft.</li>
</ul></section><section id="online-resources" class="level2"><h2 class="anchored" data-anchor-id="online-resources">Online Resources</h2>
<ul>
<li>
<a href="https://mine-cetinkaya-rundel.github.io/teach-r-online/">Teaching Statistics and Data Science Online</a> - materials for several teacher workshops taught by <a href="https://www2.stat.duke.edu/~mc301/">Mine Çetinkaya-Rundel</a>.</li>
</ul></section><section id="other" class="level2"><h2 class="anchored" data-anchor-id="other">Other</h2>
<ul>
<li>
<a href="https://bioicons.com/">Bioicons</a> - a collection of free drawings and diagrams on biological topics, which can be used in teaching (or research) presentations.</li>
</ul></section></section><section id="programming" class="level1"><h1>Programming</h1>
<section id="r-programming" class="level2"><h2 class="anchored" data-anchor-id="r-programming">R Programming</h2>
</section><section id="lists-by-others" class="level2"><h2 class="anchored" data-anchor-id="lists-by-others">Lists by others</h2>
<ul>
<li>[A list of <em>the best</em> R tools] (https://github.com/qinwf/awesome-R)</li>
</ul></section><section id="online-courses-for-r" class="level2"><h2 class="anchored" data-anchor-id="online-courses-for-r">Online Courses for R</h2>
<ul>
<li><p><a href="http://swirlstats.com/">Swirl</a> - An R package that teaches R interactively inside R. Not as polished as DataCamp, but good interactivity, great motivating feedback/messages, beginner and intermediate levels. Completely free.</p></li>
<li><p><a href="https://www.codeschool.com/courses/try-r">Try R, a free course at codeschool.com</a></p></li>
</ul></section><section id="r-books-and-tutorials" class="level2"><h2 class="anchored" data-anchor-id="r-books-and-tutorials">R books and Tutorials</h2>
<ul>
<li><p><a href="http://nathanieldphillips.com/thepiratesguidetor/">YaRrr! The Pirate’s Guide to R:</a> A nice R introduction with a piratey bend</p></li>
<li><p><a href="http://www.computerworld.com/s/article/9239625/Beginner_s_guide_to_R_Introduction">Computerworld’s R beginner guide</a></p></li>
<li><p><a href="https://www.r-bloggers.com/an-r-meta-book/">‘R Meta book’ - a list of resources on specific topics:</a></p></li>
<li><p><a href="http://www.springer.com/series/6991">Springer has a series of books called ‘Use R’</a> - A long and growing series of books on a lot of different R topics. From beginner to advanced. Many university libraries have a subscription to Springer Online, which allows you to get the books for free.</p></li>
<li><p>Roger Peng’s <a href="https://leanpub.com/rprogramming">“R programming for data science”</a> book on Leanpub. Name your own price.</p></li>
<li><p><a href="http://adv-r.had.co.nz/">“Advanced R” by Hadley Wickham for more advanced R coding</a></p></li>
</ul></section><section id="general-r-online-resources" class="level2"><h2 class="anchored" data-anchor-id="general-r-online-resources">General R Online Resources</h2>
<ul>
<li><p><a href="http://www.statmethods.net/">Quick R</a> - a great resource for many ‘how do I’ questions of common data analysis and graphics tasks.</p></li>
<li><p><a href="http://www.cookbook-r.com/">Cookbook for R</a> - another useful collection of ‘how do I do X’ information.</p></li>
<li><p><a href="https://www.r-bloggers.com/">R bloggers</a> an aggregation of blogs about all things R</p></li>
<li><p><a href="https://www.rstudio.com/resources/cheatsheets/">Convenient reference/cheat sheets for some important R tasks</a></p></li>
<li><p><a href="http://rddj.info/">Resource collection to do useful data analysis related tasks in R</a></p></li>
<li><p><a href="https://www.rdocumentation.org/">R Documentation:</a> - a webpage and R package that tries to enhance the help/documentation files for R and its packages.</p></li>
<li><p><a href="http://cran.r-project.org/web/views/">The Task View sorting R packages into categories is often useful</a></p></li>
</ul></section><section id="r-and-shiny" class="level2"><h2 class="anchored" data-anchor-id="r-and-shiny">R and Shiny</h2>
<ul>
<li><a href="http://shiny.rstudio.com/">Official shiny webpage</a></li>
<li>
<a href="https://github.com/daattali/advanced-shiny">Collection of Shiny tips and tricks</a>, <a href="https://deanattali.com/blog/advanced-shiny-tips/">see also here</a> and <a href="https://shiny.rstudio.com/articles/dynamic-ui.html">here</a>
</li>
<li><a href="https://engineering-shiny.org/">Engineering Production-Grade Shiny Apps</a></li>
<li>
<a href="https://unleash-shiny.rinterface.com/">Outstanding User Interfaces with Shiny</a> - a book covering advanced Shiny material, covering advanced styling, mobile development, integration with other web technologies, etc.</li>
</ul></section><section id="r-markdown-and-related" class="level2"><h2 class="anchored" data-anchor-id="r-markdown-and-related">R Markdown and related</h2>
<ul>
<li><a href="https://indrajeetpatil.github.io/RmarkdownTips/">Collection of R Markdown Tips and Tricks</a></li>
</ul></section><section id="r-package-development" class="level2"><h2 class="anchored" data-anchor-id="r-package-development">R package development</h2>
<ul>
<li>
<a href="http://r-pkgs.had.co.nz/">“R packages” by Hadley Wickham</a>. This (free online) book alone is enough to get a package done. Should definitely be the first (and maybe even only) source to read/use.</li>
<li><a href="http://hilaryparker.com/2014/04/29/writing-an-r-package-from-scratch/">How to quickly write an R package</a></li>
<li><a href="https://youtu.be/79s3z0gIuFU">A series of online videos talking about package development</a></li>
<li>
<code>devtools</code>, <code>roxygen2</code>, etc. - almost required for development. See Hadley’s book.</li>
<li>
<code>available</code> - tests if a specific name is still available</li>
<li>
<code>usethis</code> - helps with a lot of setup tasks for packages (and other R coding tasks)</li>
</ul></section></section><section id="data-analysis-and-statistics" class="level1"><h1>Data Analysis and Statistics</h1>
<section id="other-lists" class="level2"><h2 class="anchored" data-anchor-id="other-lists">Other lists</h2>
<ul>
<li><p><a href="https://github.com/bulutyazilim/awesome-datascience">Awesome data science</a> - Someone else’s list of resources related to data science.</p></li>
<li><p><a href="https://github.com/josephmisiti/awesome-machine-learning/blob/master/books.md">Machine learning book list</a> - A list of free online books concerning all things machine learning and data analysis.</p></li>
<li><p><a href="https://developers.google.com/machine-learning/glossary/">A machine learning list of terms and definitions.</a></p></li>
</ul></section></section><section id="online-courses-for-data-analysis" class="level1"><h1>Online Courses for Data Analysis</h1>
<ul>
<li><p><a href="https://www.coursera.org/specializations/jhu-data-science">All courses in the Data Science Specialization on Coursera</a></p></li>
<li><p><a href="https://lagunita.stanford.edu/courses/HumanitiesSciences/StatLearning/Winter2016/about">Introduction to Statistical Learning</a>.</p></li>
<li><p><a href="https://stat545.stat.ubc.ca/">Stat 545 at UBC - Data wrangling, exploration, and analysis with R</a></p></li>
<li><p><a href="http://course.fast.ai">A course on deep learning using a Python stack</a></p></li>
<li><p>Check out online providers such as Coursera, edX, Udacity, etc. for more such courses</p></li>
<li><p>Hadley taught a [Readings in applied data science] at Stanford <a href="https://github.com/hadley/stats337">and the material is online</a></p></li>
<li><p><a href="https://tinystats.github.io/teacups-giraffes-and-statistics/index.html">A very well done set of online materials teaching some basic stats concepts</a>.</p></li>
</ul></section><section id="data-analysis-books" class="level1"><h1>Data Analysis Books</h1>
<ul>
<li><p><a href="http://jsresearch.net/index.html">Introduction to Data Science</a></p></li>
<li><p><a href="https://datascienceineducation.com/">Data Science in Education</a> - all R, nice walk-through case-studies of different datasets and topics one might encounter in education.</p></li>
<li><p><a href="http://www.deeplearningbook.org/">Deep Learning book</a></p></li>
<li><p><a href="https://stat545.com/">Stat 545</a> - Jenny Bryan’s book based on her previous teaching of a course by that name (see above)</p></li>
<li><p><a href="https://theeffectbook.net/">The Effect: An Introduction to Research Design and Causality</a> - focus on causal inference.</p></li>
</ul></section><section id="specific-topics" class="level1"><h1>Specific Topics</h1>
<section id="data-cleaning-and-checking" class="level2"><h2 class="anchored" data-anchor-id="data-cleaning-and-checking">Data cleaning and checking</h2>
<ul>
<li><p>Check out the R packages tidyr, plyr, dplyr, validate, vtreat (you can find good documentation for all of them)</p></li>
<li><p><a href="http://varianceexplained.org/RData/">Data Analysis and Visualization Using R</a></p></li>
<li><p><a href="https://suzan.rbind.io/2018/01/dplyr-tutorial-1/">A nice series of blog posts describing common data manipulations for dplyr</a></p></li>
</ul></section><section id="visualization" class="level2"><h2 class="anchored" data-anchor-id="visualization">Visualization</h2>
<ul>
<li>
<a href="http://www.r-graph-gallery.com/">R Graph Gallery</a>: Has lots of examples of graphs and plots in R, including the underlying code that produced the graph. Great place to find a graph similar to one you’d like to make, take the code and adjust it.</li>
</ul></section><section id="missing-dataimputation" class="level2"><h2 class="anchored" data-anchor-id="missing-dataimputation">Missing Data/Imputation</h2>
<ul>
<li><p>Check out R packages mi, mice, Amelia, missforest, imputation</p></li>
<li><p>Little &amp; Rubin “Statistical Analysis with Missing Data, 2nd Edition”</p></li>
</ul></section><section id="p-values" class="level2"><h2 class="anchored" data-anchor-id="p-values">p-values</h2>
<ul>
<li><p><a href="http://fivethirtyeight.com/features/not-even-scientists-can-easily-explain-p-values/">Nice discussion of p-values</a></p></li>
<li><p><a href="http://fivethirtyeight.com/features/statisticians-found-one-thing-they-can-agree-on-its-time-to-stop-misusing-p-values/">Another stab at p-values</a></p></li>
<li><p><a href="https://emcrit.org/pulmcrit/p-value-cutoffs/">Another good discussion of the problem with p-values</a></p></li>
</ul></section><section id="epidemiology-specific-topics" class="level2"><h2 class="anchored" data-anchor-id="epidemiology-specific-topics">Epidemiology specific topics</h2>
<ul>
<li><p><a href="https://cran.r-project.org/package=tableone">tableone</a> - A package that allows for convenient creation of a descriptive table based on a data frame.</p></li>
<li><p><a href="https://cran.r-project.org/package=compareGroups">compareGroups</a> - A package that allows for convenient creation of result tables from linear and logistic (and related) models.</p></li>
<li><p>The <a href="https://cran.r-project.org/web/packages/survival/index.html">R survival package</a> - Lots of functionality for survival analysis.</p></li>
</ul></section><section id="time-series-analysis" class="level2"><h2 class="anchored" data-anchor-id="time-series-analysis">Time-series analysis</h2>
<ul>
<li><p>The <a href="https://cran.r-project.org/web/packages/survival/index.html">R survival package</a> - Lots of functionality for survival analysis.</p></li>
<li><p><a href="https://otexts.com/fpp2/">Forecasting: principles and practice</a></p></li>
</ul></section></section><section id="data-sources" class="level1"><h1>Data Sources</h1>
<section id="general-data-related-resources" class="level2"><h2 class="anchored" data-anchor-id="general-data-related-resources">General Data Related Resources</h2>
<ul>
<li><p>The <a href="https://labs.data.gov/dashboard/">Project Open Data Dashboard</a> gives overview statistics of available government data from various agencies.</p></li>
<li><p><a href="http://blog.kaggle.com/2016/10/21/a-guide-to-open-data-publishing-analytics/">Guide to Open Data Publishing &amp; Analytics</a> - A good article describing best practices for publishing data openly. Is also a good read for those who want to analyze other’s data.</p></li>
<li><p><a href="https://rviews.rstudio.com/2017/11/01/r-data-packages/">A short list of data related R packages</a> - packages that either access data or include data</p></li>
<li><p><a href="https://toolbox.google.com/datasetsearch">Google’s Data Search Engine</a></p></li>
</ul></section><section id="some-data-sources" class="level2"><h2 class="anchored" data-anchor-id="some-data-sources">Some data Sources</h2>
<p>(listed in no particular order)</p>
<ul>
<li><p><a href="https://www.kaggle.com/datasets">Kaggle Data</a> - A growing number of datasets used in Kaggle data analysis contests and available for any other use.</p></li>
<li><p><a href="https://data.nasdaq.com/">Nasdaq Data Link</a> - mainly finance related data</p></li>
<li><p><a href="http://www.cdc.gov/nchs/nhanes.htm">NHANES</a> - longstanding and thorough survey done by CDC</p></li>
<li><p><a href="http://seer.cancer.gov/">SEER</a> - Cancer data</p></li>
<li><p><a href="http://wonder.cdc.gov/">CDC WONDER</a> - list of mainly CDC online databases</p></li>
<li><p><a href="https://health.gov/healthypeople">Healthy People Website</a> - contains among other things links to various data sources</p></li>
<li><p><a href="https://www.hcup-us.ahrq.gov/">HCUP</a> - collection of health related databases, focusing on US wide and state-specific samples of ER and hospital visits. Not free, but not too expensive.</p></li>
<li><p><a href="https://www.clinicalstudydatarequest.com/">Clinical Study Data Request</a> - a way to get (tedious) access to clinical trial data</p></li>
<li><p><a href="https://clinicaldata.ema.europa.eu/web/cdp/home">EMA Clinical Data Portal</a> - looks like a way to get access to some clinical trial data for EMA registered studies.</p></li>
<li><p><a href="https://mimic.physionet.org/">MIMIC</a> - a free and open database of critical care patient visits to a Boston hospital.</p></li>
<li><p><a href="http://www.data.gov/">Data.gov</a> - federak government data platform.</p></li>
<li><p><a href="http://asdfree.com/">Analyze Survey Data for Free</a> - Step by Step Instructions to Explore Public Microdata from an Easy to Type Website</p></li>
<li><p><a href="https://www.cdc.gov/vaccinesafety/ensuringsafety/monitoring/vsd/accessing-data.html">Vaccine Safety Datalink (VSD)</a></p></li>
<li><p><a href="http://ghdx.healthdata.org/">Global Health Data Exchange / IHME</a></p></li>
<li><p><a href="https://vaers.hhs.gov/">Vaccine Adverse Event Reporting System (VAERS)</a></p></li>
<li><p><a href="https://nemsis.org/">National EMS Data (NEMSIS)</a></p></li>
<li><p><a href="http://data.un.org/">Data from the UN</a></p></li>
<li><p><a href="http://www.icpsr.umich.edu/">Inter-university Consortium for Political and Social Research (ICPSR)</a> - access to various social and behavioral sciences data.</p></li>
<li><p><a href="http://acrdc.gsu.edu/index.html">Federal Reserve in Atlanta Data Center</a></p></li>
<li><p><a href="https://www.healthdata.gov/">Database to search health related government data</a></p></li>
<li><p><a href="http://mran.revolutionanalytics.com/documents/data/">A list hosted by Microsoft with links to various data sources</a></p></li>
</ul></section><section id="infectious-disease-specific" class="level2"><h2 class="anchored" data-anchor-id="infectious-disease-specific">Infectious Disease Specific</h2>
<section id="general-1" class="level3"><h3 class="anchored" data-anchor-id="general-1">General</h3>
<ul>
<li>
<a href="https://www.tycho.pitt.edu/">Project Tycho</a> - infectious disease data</li>
<li>http://www.viprbrc.org</li>
<li>http://eupathdb.org</li>
<li>
<a href="https://clinepidb.org">ClinEpiDB</a> - a database of (a few) clinical epidemiology studies, focusing on infectious diseases.</li>
<li><a href="https://immport.niaid.nih.gov">ImmPort</a></li>
</ul></section><section id="influenza" class="level3"><h3 class="anchored" data-anchor-id="influenza">Influenza</h3>
<ul>
<li>http://www.fludb.org</li>
<li>WHO Flunet</li>
<li>gisaid.org</li>
<li>http://www.cdc.gov/flu/index.htm</li>
<li>http://www.ncbi.nlm.nih.gov/genomes/FLU/FLU.html</li>
<li>SystemsInfluenza.org</li>
</ul></section><section id="tb" class="level3"><h3 class="anchored" data-anchor-id="tb">TB</h3>
<ul>
<li>OTIS on CDC WONDER: http://wonder.cdc.gov/tb.html</li>
</ul></section></section></section><section id="data-and-tools-for-cancer-genomics" class="level1"><h1>Data and Tools for Cancer Genomics</h1>
<section id="data" class="level2"><h2 class="anchored" data-anchor-id="data">Data</h2>
<ul>
<li><p><a href="https://portal.gdc.cancer.gov/">The Cancer Genome Atlas (TCGA):</a> TCGA is a comprehensive collection of multi-dimensional cancer genomics data covering multiple cancer types.</p></li>
<li><p><a href="https://icgc.org/">International Cancer Genome Consortium (ICGC):</a> Description: ICGC provides high-quality genomic and clinical data from various cancer projects worldwide.</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus (GEO):</a> GEO is a public repository hosted by the National Center for Biotechnology Information (NCBI) containing a vast collection of gene expression data, including cancer datasets.</p></li>
<li><p><a href="https://ega-archive.org/">European Genome-phenome Archive (EGA):</a> Description: EGA is a repository for secure storage and sharing of human genetic and phenotypic data, including cancer datasets.</p></li>
<li><p><a href="https://gdc.cancer.gov/">National Cancer Institute (NCI) Genomic Data Commons (GDC):</a> Description: GDC is an open-access data portal providing access to a wide range of cancer genomics datasets.</p></li>
<li><p><a href="http://www.oncolnc.org/">OncoLnc</a>: Description: OncoLnc is a web resource that provides survival analysis and expression correlation for genes of interest across multiple cancer datasets.</p></li>
<li><p><a href="https://genome-cancer.ucsc.edu/">UCSC Cancer Genomics Browser:</a> The UCSC Cancer Genomics Browser offers a comprehensive collection of cancer genomics data integrated with genomic annotations.</p></li>
<li><p><a href="http://www.ilincs.org/apps/grein/">GREIN : GEO RNA-seq Experiments Interactive Navigator:</a> GREIN is an interactive web platform that provides user-friendly options to explore and analyze GEO RNA-seq data. GREIN is powered by the back-end computational pipeline for uniform processing of RNA-seq data and the large number (&gt;6,000) of already processed datasets. These datasets were retrieved from GEO and reprocessed consistently by the back-end GEO RNA-seq experiments processing pipeline (GREP2).</p></li>
</ul></section><section id="tools" class="level2"><h2 class="anchored" data-anchor-id="tools">Tools</h2>
<ul>
<li><p><a href="http://gepia2.cancer-pku.cn/">GEPIA2</a>: GEPIA2 is a web-based tool for analyzing gene expression data in cancer. It stands for Gene Expression Profiling Interactive Analysis 2 and is an updated version of the original GEPIA tool. GEPIA2 allows users to explore gene expression patterns, perform survival analyses, and visualize gene expression data across various cancer types.</p></li>
<li><p><a href="http://ualcan.path.uab.edu/">UALCAN</a>: UALCAN is a web-based platform that provides interactive and comprehensive analysis of cancer transcriptome data. It enables users to explore gene expression patterns, perform survival analyses, and compare gene expression between tumor and normal samples across different cancer types. UALCAN utilizes data from The Cancer Genome Atlas (TCGA) to facilitate cancer research and provide insights into tumor biology.</p></li>
<li><p><a href="https://www.cbioportal.org/">cBioPortal for Cancer Genomics:</a>: cBioPortal hosts a large collection of cancer genomics datasets, allowing users to explore and visualize the data.</p></li>
<li><p><a href="https://www.oncomine.org/resource/login.html">ONCOMINE</a>: ONCOMINE is a powerful web-based platform for the analysis and visualization of cancer transcriptomic data. It provides researchers with access to a vast collection of publicly available gene expression datasets derived from cancer studies. ONCOMINE allows users to explore gene expression patterns, identify potential biomarkers, and compare gene expression between different cancer types or subtypes.</p></li>
</ul></section><section id="guideline-for-bioconductor-users" class="level2"><h2 class="anchored" data-anchor-id="guideline-for-bioconductor-users">Guideline for Bioconductor Users</h2>
<p><a href="https://bioconductor.org/">Bioconductor</a> is an open-source and open-development software project that provides a comprehensive collection of bioinformatics and computational biology tools in the R programming language. It focuses on the analysis and comprehension of high-throughput genomic data, including DNA sequencing, RNA sequencing, microarray analysis, proteomics, and more.</p>
</section><section id="required-software" class="level2"><h2 class="anchored" data-anchor-id="required-software">Required software</h2>
<ul>
<li>R: <a href="http://www.r-project.org/" class="uri">http://www.r-project.org/</a> (FREE)</li>
<li>RStudio (additional libraries required): <a href="http://www.rstudio.com/" class="uri">http://www.rstudio.com/</a> (FREE)</li>
</ul></section><section id="prework" class="level2"><h2 class="anchored" data-anchor-id="prework">Prework</h2>
<p>Before attending the any workshop please have the following installed and configured on your machine. - Recent version of <strong>R</strong> - Recent version of <strong>RStudio</strong> - Most recent release of the <a href="https://www.bioconductor.org/install/"><strong>Bioconductor</strong></a> and other packages used in courses</p>
<p>Install the latest release of R, then get the latest version of Bioconductor by starting R and entering the commands.</p>
<div class="cell">
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html">require</a></span><span class="op">(</span><span class="st"><a href="https://bioconductor.github.io/BiocManager/">"BiocManager"</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html">install</a></span><span class="op">(</span>version <span class="op">=</span> <span class="st">"3.16"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li>
<p>Ensure you can knit R markdown documents</p>
<ul>
<li>Open RStudio and create a new Rmarkdown document</li>
<li>Save the document and check you are able to knit it.</li>
</ul>
</li>
</ul></section><section id="install-bioconductor-packages" class="level2"><h2 class="anchored" data-anchor-id="install-bioconductor-packages">Install Bioconductor Packages</h2>
<div class="cell">
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html">require</a></span><span class="op">(</span><span class="st"><a href="https://bioconductor.github.io/BiocManager/">"BiocManager"</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html">install</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Install specific packages, e.g., “GenomicFeatures” and “AnnotationDbi”, with</p>
<div class="cell">
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html">install</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"GenomicFeatures"</span>, <span class="st">"AnnotationDbi"</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The <code><a href="https://bioconductor.github.io/BiocManager/reference/install.html">install()</a></code> function (in the BiocManager package) has arguments that change its default behavior; type ?install for further help.</p>
</section><section id="r-packages-rnaseq-and-single-cell-rna-seq-analysis" class="level2"><h2 class="anchored" data-anchor-id="r-packages-rnaseq-and-single-cell-rna-seq-analysis">R Packages RNASeq and Single-cell RNA-seq Analysis</h2>
<ul>
<li>
<code>DESeq2</code>: <a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a> is a widely used package for differential gene expression analysis in RNA-seq data.</li>
<li>
<code>edgeR</code>: <a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a> is another popular package for differential gene expression analysis in RNA-seq data.</li>
<li>
<code>limma</code>: <a href="https://bioconductor.org/packages/release/bioc/html/limma.html">limma</a> is a package commonly used for the analysis of microarray and RNA-seq data, particularly for differential expression analysis.</li>
<li>
<code>Ballgown</code>: <a href="https://bioconductor.org/packages/release/bioc/html/ballgown.html">Ballgown</a> is a package for differential expression analysis and visualization of transcriptome assembly data.</li>
<li>
<code>DEXSeq</code>: <a href="https://bioconductor.org/packages/release/bioc/html/DEXSeq.html">DEXSeq</a> is specifically designed for the detection of differential exon usage in RNA-seq data.</li>
<li>
<code>NOISeq</code>: <a href="https://bioconductor.org/packages/release/bioc/html/NOISeq.html">NOISeq</a> is a package for non-parametric analysis of differential expression in RNA-seq data.</li>
<li>
<code>clusterProfiler</code>: <a href="https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html">clusterProfiler</a> is a package for functional enrichment analysis of gene clusters derived from RNA-seq data.</li>
<li>
<code>GenomicFeatures</code>: <a href="https://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html">GenomicFeatures</a> provides tools for working with genomic features, such as gene models, and is useful for annotating RNA-seq data.</li>
<li>
<code>Seurat</code>: <a href="https://satijalab.org/seurat/">Seurat</a> is a package for single-cell RNA-seq data analysis, allowing exploration and visualization of cellular heterogeneity.</li>
</ul></section><section id="blogs-for-r-programming-statistics-and-data-analyis" class="level2"><h2 class="anchored" data-anchor-id="blogs-for-r-programming-statistics-and-data-analyis">Blogs for R Programming, Statistics, and Data Analyis</h2>
<ul>
<li>Programiz - <a href="https://www.datamentor.io/r-programming/">https://www.datamentor.io/r-programming/</a>
</li>
<li>PennState STAT 484 - <a href="https://online.stat.psu.edu/stat484/">https://online.stat.psu.edu/stat484/</a>
</li>
<li>PennState Topics in R Statistical Language - <a href="https://online.stat.psu.edu/stat484/">https://online.stat.psu.edu/stat484/</a>
</li>
<li>Simply Statistics - <a href="https://simplystatistics.org/">https://simplystatistics.org/</a>
</li>
<li>TutorialPoint - <a href="https://www.tutorialspoint.com/r/index.htm">https://www.tutorialspoint.com/r/index.htm</a>
</li>
<li>R for Biologists - <a href="https://www.rforbiologists.org/">https://www.rforbiologists.org/</a>
</li>
<li>Computational Genomics with R - <a href="https://compgenomr.github.io/book/">https://compgenomr.github.io/book/</a>
</li>
<li>Stat and R - <a href="https://statsandr.com/">https://statsandr.com/</a>
</li>
<li>Rafa Lab - <a href="https://rafalab.github.io/pages/harvardx.html">https://rafalab.github.io/pages/harvardx.html</a>
</li>
<li>University of Florida - <a href="https://bolt.mph.ufl.edu/software/r-phc-6055/">https://bolt.mph.ufl.edu/software/r-phc-6055/</a>
</li>
</ul></section><section id="videos" class="level2"><h2 class="anchored" data-anchor-id="videos">Videos</h2>
<ul>
<li>
<p><strong>July 9, 2023</strong>: I’m thrilled to announce that the highly anticipated videos from our workshop on “R for Research: Fundamentals of R - Part 1” are now available for viewing. Whether you missed the live event or want to revisit the valuable insights shared during the session, these videos are your gateway to mastering the basics of R programming language for data analysis and research. Join us as we explore the foundations of R and learn essential skills to enhance your data analysis capabilities. Don’t wait any longer; dive into the videos today and take your research to new heights! Check it out: </p>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/dQhZjmw9BK0" title="R for Research || Fundamentals of R: Part 1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
</li>
<li>
<p><strong>April 12, 2023</strong>: Watch this informative 2-hour workshop on how <strong>NASA Earth Observing Data</strong> can help improve public health. Discover how we can use these data to monitor our environment and identify potential health risks. Learn about the different ways NASA Earth Observing Data can benefit our communities and keep us safe. Check it out: </p>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/rHJUhAz0_98" title="Using NASA Earth Observing Data for Public Health" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
</li>
</ul>


</section></section></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer"><div class="nav-footer">
    <div class="nav-footer-left">
<p>© Copyright 2023, Preonath Chondrow Dev</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
<li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/PreonathShuvo" rel="me">
      <i class="bi bi-twitter" role="img" aria-label="twitter">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/preonath" rel="me">
      <i class="bi bi-github" role="img" aria-label="github">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.youtube.com/@preonath" rel="me">
      <i class="bi bi-youtube" role="img" aria-label="youtube">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/preonath-shuvo-26aa1416b/">
      <i class="bi bi-linkedin" role="img" aria-label="linkedin">
</i> 
    </a>
  </li>  
</ul>
</div>
  </div>
</footer>


</body></html>