<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.555">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="description" content="Personal Website for Preonath Chondrow Dev (Bioinformatician, Data Analyst)">

<title>What I Use | Preonath</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href=".//files/profiles/icon.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<meta property="og:title" content="What I Use | Preonath">
<meta name="twitter:title" content="What I Use | Preonath">
<meta name="twitter:card" content="summary">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./research.html"> 
<span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./publications.html"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./conference.html"> 
<span class="menu-text">Conference</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./training.html"> 
<span class="menu-text">Training</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./teaching.html"> 
<span class="menu-text">Teaching</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./use.html" aria-current="page"> 
<span class="menu-text">What I Use</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./cv/index.html"> 
<span class="menu-text">CV</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content column-page" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">What I Use</h1>
</div>



<div class="quarto-title-meta column-page">

    
  
    
  </div>
  


</header>


<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true">Technical Skills</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">R</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">Python</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false">Bash</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-5" role="tab" aria-controls="tabset-1-5" aria-selected="false">Data Resources</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-6" role="tab" aria-controls="tabset-1-6" aria-selected="false">Genomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-7-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-7" role="tab" aria-controls="tabset-1-7" aria-selected="false">Metagenomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-8-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-8" role="tab" aria-controls="tabset-1-8" aria-selected="false">Teaching Tools</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<!-- ## Literature Review  -->
<!-- - [typeset.io](https://typeset.io/) - Typeset.io is a platform designed to assist researchers, academics, and authors in the process of writing, formatting, and preparing documents for publication.  -->
<!-- - [scholarcy](https://www.scholarcy.com/) - The AI-powered article summarizer.  -->
<!-- - [elicit](https://elicit.org/) - AI-based research assistant.  -->
<!-- ## Journal Finders  -->
<!-- - [Journal Finder by Elsevier](https://journalfinder.elsevier.com/): This tool allows you to enter your manuscript title and abstract to find journals that are a good match for your research. It also provides information on journal metrics and submission guidelines. -->
<!-- - [Springer Journal Suggester](https://journalsuggester.springer.com/): Springer's tool helps you find the right journal for your research by analyzing your abstract or keywords and suggesting journals that publish articles in your field. -->
<!-- - [PubMed Journal Selector](https://www.ncbi.nlm.nih.gov/pmc/journals/): If your research is in the biomedical or life sciences, PubMed's Journal Selector can assist you in finding journals that match your keywords and research area. -->
<!-- - [JANE (Journal/Author Name Estimator)](https://jane.biosemantics.org/): JANE is a free online tool that helps you find journals and authors based on the text of your article's title and abstract. -->
<!-- - [JournalGuide](https://www.journalguide.com/): This tool provides a comprehensive database of journals in various fields and allows you to search for journals by keywords or browse by subject area. -->
<!-- - [Scopus Journal Finder](https://www.scopus.com/sources.uri): Scopus, a bibliographic database, offers a journal finder feature that helps you find journals related to your research area based on keywords or article titles. -->
<!-- - [DOAJ (Directory of Open Access Journals)](https://doaj.org/): If you're interested in open-access journals, DOAJ is a directory of freely available scholarly journals that you can search by subject or keyword. -->
<!-- - [Scimago Journal & Country Rank](https://www.scimagojr.com/): On the Scimago platform, you can find information about journals, their rankings, citation data, and more, which can be useful for researchers looking to identify suitable journals for their research or assess the impact and prestige of journals in their field. It's a valuable tool for academic research and evaluation. -->
<p>Languages: <strong>Bash, Python, C, R, Java</strong></p>
<p>Database Management System(<strong>SQL, MySQL, Redis</strong>)</p>
<p>Skillful at <strong>Machine learning and Deep Learning</strong>.</p>
<p>Data analysis using <strong>Numpy, Pandas, Malplotlib, Seaborn, Keras, SciKit-Learn TensorFlow.</strong></p>
<p>Microarray analysis.</p>
<p>RNA sequencing Analysis</p>
<p>Single Cell RNA sequencing Analysis</p>
<p>Molecular biology:</p>
<ul>
<li>Agarose &amp; Polyacrylamide Gel Electrophoresis and Imaging</li>
<li>Blotting techniques</li>
<li>DNA &amp; RNA extraction</li>
<li>Molecular cloning &amp; Restriction enzymes</li>
<li>Conventional and Real-time PCR</li>
<li>ELISA &amp; ICT</li>
<li>Vaccine trial &amp; microsurgery on the mouse model.</li>
<li>Library preparation and pulling for Bacterial and Viral sequencing of Illumina</li>
<li>Library preparation and pulling of Nanopore.</li>
</ul>
<p>Microbiology:</p>
<ul>
<li>Culture media preparation (Blood, MacCONKEY, Muller-Hinton blood, and Chocolate agar) &amp; Gram Staining.</li>
<li>Catalase, Coagulase, Optochine sensitivity, CAMP, Bile solubility, Oxidase, Biochemical(TSI, MIU, - - Citrate), Satellitism, X-V factor, Antibiotic susceptibility test.</li>
</ul>
<!-- # machine learning and deep learning tools  -->
<!-- **Bash :** -->
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<section id="data-wrangling" class="level2">
<h2 class="anchored" data-anchor-id="data-wrangling">Data Wrangling</h2>
<ul>
<li><a href="https://readxl.tidyverse.org/">readxl</a> for importing data into R</li>
<li>dplyr, tidyr and others from the <a href="https://www.tidyverse.org/">tidyverse</a> for data preparation.</li>
</ul>
</section>
<section id="data-visualization" class="level2">
<h2 class="anchored" data-anchor-id="data-visualization">Data Visualization</h2>
<ul>
<li><a href="https://ggplot2.tidyverse.org/">ggplot2</a> for the vast majority of the graphics, together with the <a href="https://github.com/hrbrmstr/hrbrthemes">hrbrtheme</a> for styling.</li>
<li><a href="https://github.com/thomasp85/patchwork">patchwork</a> to put graphics together.</li>
<li><a href="https://github.com/thomasp85/ggraph">ggraph</a> and igraph for most of the network related graphics</li>
<li><a href="https://r-graph-gallery.com/interactive-charts">plotly</a> and other html widgets for interactive graphics.</li>
<li><a href="https://github.com/cran/RColorBrewer">RColorBrewer</a> and <a href="https://github.com/sjmgarnier/viridis">viridis</a> and colormap to control color in charts.</li>
<li><a href="https://github.com/slowkow/ggrepel">Ggrepel</a> and other ggplot2 extension that make your life simpler.</li>
<li><a href="https://github.com/talgalili/heatmaply">Heatmaply</a> for most of the heatmaps</li>
</ul>
</section>
<section id="reproducible-research" class="level2">
<h2 class="anchored" data-anchor-id="reproducible-research">Reproducible Research</h2>
<ul>
<li><a href="https://rmarkdown.rstudio.com/">R Markdown</a> to produce statistical reports.</li>
<li><a href="https://quarto.org/">Quarto</a> to build 95% of the website for my courses and others.</li>
</ul>
</section>
<section id="statistical-modeling" class="level2">
<h2 class="anchored" data-anchor-id="statistical-modeling">Statistical Modeling</h2>
<!-- - [easystats](https://easystats.github.io/easystats/) for easy statistical modeling, visualization, and reporting  -->
<p>Static modelling in SPSS entails building and evaluating models that represent relationships within a dataset at a particular point in time. ANOVA, descriptive statistics, and regression analysis are common methods. Data preparation, variable selection, model construction, and evaluation are all steps in the process. Coefficients, p-values, and R-squared are used in output interpretation to determine the significance and strength of relationships. Applications include business analytics, social sciences, and market research. It is crucial to take into account presumptions and constraints, such as the presumption of ongoing relationships. Static modelling helps with decision-making based on observed patterns by offering insights into cross-sectional data.</p>
</section>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<ul>
<li><a href="https://numpy.org/doc/stable/index.html">NumPy</a> for scientific computing.</li>
<li><a href="https://pandas.pydata.org/">Pandas</a> for data wrangling and analysis</li>
<li><a href="https://matplotlib.org/">Matplotlib</a> for data visulization</li>
<li><a href="https://seaborn.pydata.org/">Seaborn</a> for advance statistical visualizations</li>
<li><a href="https://plotly.com/python/">Plotly</a> for interative data visualization</li>
<li><a href="https://researchpy.readthedocs.io/en/latest/">researchpy</a> to summarize data and perform statistical tests.</li>
<li><a href="https://www.dask.org/">Dask</a> for big data analysis</li>
<li><a href="https://scikit-learn.org/stable/">scikit-learn</a> for machine learning</li>
<li><a href="https://scikit-image.org/">scikit-image</a> for life science image manipulation</li>
</ul>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<p><strong>Bash :</strong></p>
</div>
<div id="tabset-1-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-5-tab">
<section id="guides" class="level2">
<h2 class="anchored" data-anchor-id="guides">Guides</h2>
<ul>
<li><p>The <a href="https://labs.data.gov/dashboard/">Project Open Data Dashboard</a> gives overview statistics of available government data from various agencies.</p></li>
<li><p><a href="http://blog.kaggle.com/2016/10/21/a-guide-to-open-data-publishing-analytics/">Guide to Open Data Publishing &amp; Analytics</a> - A good article describing best practices for publishing data openly. Is also a good read for those who want to analyze other’s data.</p></li>
<li><p><a href="https://rviews.rstudio.com/2017/11/01/r-data-packages/">A short list of data related R packages</a> - packages that either access data or include data</p></li>
<li><p><a href="https://toolbox.google.com/datasetsearch">Google’s Data Search Engine</a></p></li>
</ul>
</section>
<section id="some-data-sources" class="level2">
<h2 class="anchored" data-anchor-id="some-data-sources">Some Data Sources</h2>
<ul>
<li><p><a href="https://www.kaggle.com/datasets">Kaggle Data</a> - A growing number of datasets used in Kaggle data analysis contests and available for any other use.</p></li>
<li><p><a href="https://data.nasdaq.com/">Nasdaq Data Link</a> - mainly finance related data</p></li>
<li><p><a href="http://www.cdc.gov/nchs/nhanes.htm">NHANES</a> - longstanding and thorough survey done by CDC</p></li>
<li><p><a href="http://seer.cancer.gov/">SEER</a> - Cancer data</p></li>
<li><p><a href="http://wonder.cdc.gov/">CDC WONDER</a> - list of mainly CDC online databases</p></li>
<li><p><a href="https://health.gov/healthypeople">Healthy People Website</a> - contains among other things links to various data sources</p></li>
<li><p><a href="https://www.hcup-us.ahrq.gov/">HCUP</a> - collection of health related databases, focusing on US wide and state-specific samples of ER and hospital visits. Not free, but not too expensive.</p></li>
<li><p><a href="https://www.clinicalstudydatarequest.com/">Clinical Study Data Request</a> - a way to get (tedious) access to clinical trial data</p></li>
<li><p><a href="https://clinicaldata.ema.europa.eu/web/cdp/home">EMA Clinical Data Portal</a> - looks like a way to get access to some clinical trial data for EMA registered studies.</p></li>
<li><p><a href="https://mimic.physionet.org/">MIMIC</a> - a free and open database of critical care patient visits to a Boston hospital.</p></li>
<li><p><a href="http://www.data.gov/">Data.gov</a> - federak government data platform.</p></li>
<li><p><a href="http://asdfree.com/">Analyze Survey Data for Free</a> - Step by Step Instructions to Explore Public Microdata from an Easy to Type Website</p></li>
<li><p><a href="https://www.cdc.gov/vaccinesafety/ensuringsafety/monitoring/vsd/accessing-data.html">Vaccine Safety Datalink (VSD)</a></p></li>
<li><p><a href="http://ghdx.healthdata.org/">Global Health Data Exchange / IHME</a></p></li>
<li><p><a href="https://vaers.hhs.gov/">Vaccine Adverse Event Reporting System (VAERS)</a></p></li>
<li><p><a href="https://nemsis.org/">National EMS Data (NEMSIS)</a></p></li>
<li><p><a href="http://data.un.org/">Data from the UN</a></p></li>
<li><p><a href="http://www.icpsr.umich.edu/">Inter-university Consortium for Political and Social Research (ICPSR)</a> - access to various social and behavioral sciences data.</p></li>
<li><p><a href="http://acrdc.gsu.edu/index.html">Federal Reserve in Atlanta Data Center</a></p></li>
<li><p><a href="https://www.healthdata.gov/">Database to search health related government data</a></p></li>
<li><p><a href="http://mran.revolutionanalytics.com/documents/data/">A list hosted by Microsoft with links to various data sources</a></p></li>
</ul>
</section>
<section id="infectious-disease-specific" class="level2">
<h2 class="anchored" data-anchor-id="infectious-disease-specific">Infectious Disease Specific</h2>
<section id="general" class="level3">
<h3 class="anchored" data-anchor-id="general">General</h3>
<ul>
<li><a href="https://www.tycho.pitt.edu/">Project Tycho</a> - infectious disease data</li>
<li><a href="http://www.viprbrc.org" class="uri">http://www.viprbrc.org</a></li>
<li><a href="http://eupathdb.org" class="uri">http://eupathdb.org</a></li>
<li><a href="https://clinepidb.org">ClinEpiDB</a> - a database of (a few) clinical epidemiology studies, focusing on infectious diseases.</li>
<li><a href="https://immport.niaid.nih.gov">ImmPort</a></li>
</ul>
</section>
<section id="influenza" class="level3">
<h3 class="anchored" data-anchor-id="influenza">Influenza</h3>
<ul>
<li>http://www.fludb.org</li>
<li><who flunet=""></who></li>
<li>&lt;gisaid.org&gt;</li>
<li><a href="http://www.cdc.gov/flu/index.htm" class="uri">http://www.cdc.gov/flu/index.htm</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/genomes/FLU/FLU.html" class="uri">http://www.ncbi.nlm.nih.gov/genomes/FLU/FLU.html</a></li>
<li><a href="http://www.SystemsInfluenza.org" class="uri">http://www.SystemsInfluenza.org</a></li>
</ul>
</section>
<section id="tb" class="level3">
<h3 class="anchored" data-anchor-id="tb">TB</h3>
<ul>
<li>OTIS on CDC WONDER: <a href="http://wonder.cdc.gov/tb.html" class="uri">http://wonder.cdc.gov/tb.html</a></li>
</ul>
</section>
</section>
</div>
<div id="tabset-1-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-6-tab">
<section id="data-sources" class="level2">
<h2 class="anchored" data-anchor-id="data-sources">Data Sources</h2>
<ul>
<li><a href="https://portal.gdc.cancer.gov/">The Cancer Genome Atlas (TCGA):</a> TCGA is a comprehensive collection of multi-dimensional cancer genomics data covering multiple cancer types.</li>
<li><a href="https://icgc.org/">International Cancer Genome Consortium (ICGC):</a> Description: ICGC provides high-quality genomic and clinical data from various cancer projects worldwide.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus (GEO):</a> GEO is a public repository hosted by the National Center for Biotechnology Information (NCBI) containing a vast collection of gene expression data, including cancer datasets.</li>
<li><a href="https://ega-archive.org/">European Genome-phenome Archive (EGA):</a> Description: EGA is a repository for secure storage and sharing of human genetic and phenotypic data, including cancer datasets.</li>
<li><a href="https://gdc.cancer.gov/">National Cancer Institute (NCI) Genomic Data Commons (GDC):</a> Description: GDC is an open-access data portal providing access to a wide range of cancer genomics datasets.</li>
</ul>
</section>
<section id="analysis-tools" class="level2">
<h2 class="anchored" data-anchor-id="analysis-tools">Analysis Tools</h2>
<ul>
<li><p><a href="http://gepia2.cancer-pku.cn/">GEPIA2</a>: GEPIA2 is a web-based tool for analyzing gene expression data in cancer. It stands for Gene Expression Profiling Interactive Analysis 2 and is an updated version of the original GEPIA tool. GEPIA2 allows users to explore gene expression patterns, perform survival analyses, and visualize gene expression data across various cancer types.</p></li>
<li><p><a href="http://timer.cistrome.org/">TIMER2.0</a>: TIMER is a comprehensive resource for systematical analysis of immune infiltrates across diverse cancer types. This version of webserver provides immune infiltrates’ abundances estimated by multiple immune deconvolution methods, and allows users to generate high-quality figures dynamically to explore tumor immunological, clinical and genomic features comprehensively.</p></li>
<li><p><a href="http://ualcan.path.uab.edu/">UALCAN</a>: UALCAN is a web-based platform that provides interactive and comprehensive analysis of cancer transcriptome data. It enables users to explore gene expression patterns, perform survival analyses, and compare gene expression between tumor and normal samples across different cancer types. UALCAN utilizes data from The Cancer Genome Atlas (TCGA) to facilitate cancer research and provide insights into tumor biology.</p></li>
<li><p><a href="https://www.cbioportal.org/">cBioPortal for Cancer Genomics:</a>: cBioPortal hosts a large collection of cancer genomics datasets, allowing users to explore and visualize the data.</p></li>
<li><p><a href="http://www.ilincs.org/apps/grein/">GREIN : GEO RNA-seq Experiments Interactive Navigator:</a> GREIN is an interactive web platform that provides user-friendly options to explore and analyze GEO RNA-seq data. GREIN is powered by the back-end computational pipeline for uniform processing of RNA-seq data and the large number (&gt;6,000) of already processed datasets. These datasets were retrieved from GEO and reprocessed consistently by the back-end GEO RNA-seq experiments processing pipeline (GREP2).</p></li>
<li><p><a href="http://www.oncolnc.org/">OncoLnc</a>: Description: OncoLnc is a web resource that provides survival analysis and expression correlation for genes of interest across multiple cancer datasets.</p></li>
<li><p><a href="https://genome-cancer.ucsc.edu/">UCSC Cancer Genomics Browser:</a> The UCSC Cancer Genomics Browser offers a comprehensive collection of cancer genomics data integrated with genomic annotations.</p></li>
<li><p><a href="https://www.oncomine.org/resource/login.html">ONCOMINE</a>: ONCOMINE is a powerful web-based platform for the analysis and visualization of cancer transcriptomic data. It provides researchers with access to a vast collection of publicly available gene expression datasets derived from cancer studies. ONCOMINE allows users to explore gene expression patterns, identify potential biomarkers, and compare gene expression between different cancer types or subtypes.</p></li>
</ul>
</section>
<section id="r-packages" class="level2">
<h2 class="anchored" data-anchor-id="r-packages">R Packages</h2>
<ul>
<li><p><a href="https://bioconductor.org/packages/release/bioc/vignettes/TCGAbiolinks/inst/doc/index.html#Required_libraries">TCGAbiolinks</a>: An R/Bioconductor package for integrative analysis with GDC data. TCGAbiolinks is able to access The National Cancer Institute (NCI) Genomic Data Commons (GDC) thorough its GDC Application Programming Interface (API) to search, download and prepare relevant data for analysis in R.</p></li>
<li><p><a href="https://bioconductor.org/packages/release/bioc/vignettes/maftools/inst/doc/maftools.html">maftools</a>: Summarize, Analyze and Visualize MAF Files. This package attempts to summarize, analyze, annotate and visualize MAF files in an efficient manner from either TCGA sources or any in-house studies as long as the data is in MAF format.</p></li>
<li><p><a href="https://bioconductor.org/packages/release/bioc/vignettes/SummarizedExperiment/inst/doc/SummarizedExperiment.html">SummarizedExperiment</a>: The SummarizedExperiment container contains one or more assays, each represented by a matrix-like object of numeric or other mode. The rows typically represent genomic ranges of interest and the columns represent samples.</p></li>
<li><p><a href="https://bioconductor.org/packages/release/bioc/vignettes/MutationalPatterns/inst/doc/Introduction_to_MutationalPatterns.html">MutationalPatterns</a>: Comprehensive genome-wide analysis of mutational processes. he package covers a wide range of patterns including: mutational signatures, transcriptional and replicative strand bias, lesion segregation, genomic distribution and association with genomic features, which are collectively meaningful for studying the activity of mutational processes. The package works with single nucleotide variants (SNVs), insertions and deletions (Indels), double base substitutions (DBSs) and larger multi base substitutions (MBSs). The package provides functionalities for both extracting mutational signatures de novo and determining the contribution of previously identified mutational signatures on a single sample level. MutationalPatterns integrates with common R genomic analysis workflows and allows easy association with (publicly available) annotation data.</p></li>
</ul>
</section>
</div>
<div id="tabset-1-7" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-7-tab">
<p><strong>QIIME2 :</strong> Instead of taking changes over time into account, static modelling in SPSS entails building and evaluating models that represent relationships within a dataset at a particular point in time. ANOVA, descriptive statistics, and regression analysis are common methods. Data preparation, variable selection, model construction, and evaluation are all steps in the process. Coefficients, p-values, and R-squared are used in output interpretation to determine the significance and strength of relationships. Applications include business analytics, social sciences, and market research. It is crucial to take into account presumptions and constraints, such as the presumption of ongoing relationships. Static modelling helps with decision-making based on observed patterns by offering insights into cross-sectional data.</p>
</div>
<div id="tabset-1-8" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-8-tab">
<ul>
<li><p><strong>Bioinformatician</strong> -DNA, RNA extraction, PCR, Library Preparation for Illumina and Nanopore, Tapstation, Sample sheet preparation.</p></li>
<li><p><strong>Illumina</strong> Converting(BCL2fastq), Quality checking(fastQC, MultiQC, Quast), Quality control(Trimmomatic), Assembly(Unicycler, Spades, Megahit)</p></li>
<li><p><strong>Annotation</strong> Kraken2 , Prokka, Seroba, AMRFinderPlus, Abricat, SRST, MLST, Snippy, Mafft, fasta2phylip, Raxml-ng, Poppunk, PlasmidFinder, ResFinder, Blast, Pharokka. . <!-- - [learnr](https://rstudio.github.io/learnr/) - R package that allows development of interactive web-based R tutorials. --></p></li>
<li><p><strong>Research Fellow</strong> Develop ML based pipeline for shot-gun sequence, 16s rrna sequencing analysis using qiime2, RNA seq using DESeq2, stats, ggplot, corrplot, pheatmap, EDASeq, gProfileR</p></li>
</ul>
</div>
</div>
</div>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>© Copyright 2023, Preonath Chondrow Dev</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/PreonathShuvo" rel="me">
      <i class="bi bi-twitter" role="img" aria-label="twitter">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/preonath" rel="me">
      <i class="bi bi-github" role="img" aria-label="github">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.youtube.com/@preonath" rel="me">
      <i class="bi bi-youtube" role="img" aria-label="youtube">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/preonath-shuvo-26aa1416b/">
      <i class="bi bi-linkedin" role="img" aria-label="linkedin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>